src/io/RELION5/RELION5ParticleData.py

```diff
@@ -0,0 +1,205 @@
+# vim: set expandtab shiftwidth=4 softtabstop=4:
+
+# General
+import numpy as np
+import starfile
+import pandas as pd
+
+# Chimerax
+from chimerax.core.errors import UserError
+
+# This package
+from ..formats import ArtiaXFormat
+from ..ParticleData import ParticleData, EulerRotation
+from ...widgets.Relion5SaveAddInfo import CoordInputDialog
+
+EPSILON = np.finfo(np.float32).eps
+EPSILON16 = 16 * EPSILON
+
+class RELIONEulerRotation(EulerRotation):
```

Comment:
Since this is probably the same class as in the regular RELION particle data, we should import it here to avoid code duplication.